README.md

```diff
@@ -82,37 +82,54 @@ These are the empty repositories that have PRs waiting to be merged.
+  safe wrappers.
+- [vm-allocator](https://github.com/rust-vmm/vm-allocator): abstractions for
+  allocating resources needed by the VMM while running (such as GSI numbers,
+  MMIO ranges).
+- [vm-virtio](https://github.com/rust-vmm/vm-virtio): workspace for virtio
+  devices and related helpers.
 - [vmm-reference](https://github.com/rust-vmm/vmm-reference): reference VMM
   built solely with `rust-vmm` crates and minimal glue.
+- [vhost-device](https://github.com/rust-vmm/vhost-device): A workspace for
+  devices implemented using the
+  [vhost-user-backend](https://github.com/rust-vmm/vhost-user-backend).
 
 ### crates.io
 
+- [event-manager](https://crates.io/crates/event-manager): abstractions
+  for implementing event based systems.
 - [kvm-bindings](https://crates.io/crates/kvm-bindings): Rust FFI bindings
   to KVM generated using [bindgen](https://crates.io/crates/bindgen).
 - [kvm-ioctls](https://crates.io/crates/kvm-ioctls): Safe wrappers over the
   KVM API.
-- [vfio-bindings](https://crates.io/crates/vfio-bindings): Rust FFI
-  bindings for using the VFIO framework.
+- [linux-loader](https://crates.io/crates/linux-loader): parser and loader
+  for vmlinux and bzImage images as well as some other helpers for kernel
+  commandline.
+- [seccompiler](https://github.com/rust-vmm/seccompiler): Linux seccomp-bpf
+  jailing utilities.
+- [vfio-bindings](https://github.com/rust-vmm/vfio/tree/main/crates/vfio-bindings):
+  Rust FFI bindings for using the VFIO framework.
+- [vfio-ioctls](https://github.com/rust-vmm/vfio/tree/main/crates/vfio-ioctls):
+  safe wrappers over the VFIO framework.
 - [vhost](https://github.com/rust-vmm/vhost): a crate to support vhost backend
   drivers for virtio devices.
+- [vhost-user-backend](https://github.com/rust-vmm/vhost-user-backend): provides
+  a framework to implement vhost-user backend services
 - [virtio-bindings](https://crates.io/crates/virtio-bindings): Rust FFI
   bindings to virtio kernel headers generated using
   [bindgen](https://crates.io/crates/bindgen).
+- [virtio-queue](https://github.com/rust-vmm/vm-virtio/tree/main/crates/virtio-queue):
+  virtio queue implementation.
+- [vm-device](https://github.com/rust-vmm/vm-device): a virtual machine device
+  model crate.
+- [vm-fdt](https://github.com/rust-vmm/vm-fdt): a Flattened Device Tree writer.
 - [vm-memory](https://crates.io/crates/vm-memory): abstractions over a
   virtual machine's memory.
-- [vmm-sys-util](https://crates.io/crates/vmm-sys-util/): collection of
-  modules providing helpers and utilities for building VMMs and hypervisors.
-- [event-manager](https://crates.io/crates/event-manager): abstractions
-  for implementing event based systems.
-- [linux-loader](https://crates.io/crates/linux-loader): parser and loader
-  for vmlinux and bzImage images as well as some other helpers for kernel
-  commandline.
 - [vm-superio](https://crates.io/crates/vm-superio): emulation for legacy
```

Comment:
I think we should add vm-superio-ser as well.

Reply:
Done.